client/src/components/layout/chat-room/chat-room-message-input.tsx

```
@@ -1,25 +1,29 @@
-	"data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyNCIgaGVpZ2h0PSIyNCIgdmlld0JveD0iMCAwIDI0IDI0IiBmaWxsPSJub25lIiBzdHJva2U9IndoaXRlIiBzdHJva2Utd2lkdGg9IjEiIHN0cm9rZS1saW5lY2FwPSJyb3VuZCIgc3Ryb2tlLWxpbmVqb2luPSJyb3VuZCIgY2xhc3M9Imx1Y2lkZSBsdWNpZGUtcGFwZXJjbGlwIj48cGF0aCBkPSJNMTMuMjM0IDIwLjI1MiAyMSAxMi4zIi8+PHBhdGggZD0ibTE2IDYtOC40MTQgOC41ODZhMiAyIDAgMCAwIDAgMi44MjggMiAyIDAgMCAwIDIuODI4IDBsOC40MTQtOC41ODZhNCA0IDAgMCAwIDAtNS42NTYgNCA0IDAgMCAwLTUuNjU2IDBsLTguNDE1IDguNTg1YTYgNiAwIDEgMCA4LjQ4NiA4LjQ4NiIvPjwvc3ZnPg==";
+const ErrorMessages = ({ errors }: { errors: FieldErrors<MessageInput> }) => (
+	<div className="flex flex-col">
+		{Object.keys(errors).map((error) => (
+			<p key={error} className="text-red-400 text-xs">
+				{errors[error as keyof MessageInput]?.message as string}
+			</p>
+		))}
+	</div>
+);
 
 export function MessageInput({ chatId, chatName, isPrivateChat }: ChatInputProps) {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'MessageInput'. Consider to delete it or rename it.

'MessageInput' is defined here:


(lint/suspicious/noRedeclare)